dart/lib/src/http_client/failed_request_client.dart

```diff
@@ -65,10 +65,7 @@ import 'sentry_http_client.dart';
 /// ```
 class FailedRequestClient extends BaseClient {
   FailedRequestClient({
-    this.maxRequestBodySize = MaxRequestBodySize.never,
```

Comment:
`dio/example` still contains `maxRequestBodySize`

Reply:
I'm not sure what to do with the example, 
`dio.addSentry();` should it be like this? or should this line be completely removed?

Reply:
`dio.addSentry()` should be kept, but the optional parameters of that function can be read from SentryOptions. So you can remove them from the parameter list of that function.